en/document-v1-api-guide.html

```diff
@@ -112,8 +115,14 @@ <h2 id="request-examples">Request examples</h2>
 </tr><tr id="delete">
   <th>DELETE</th>
   <td>
+    Delete document with ID 1:
 <pre>
 $ curl -X DELETE http://hostname:8080/document/v1/namespace/music/docid/1
+</pre>
+    Delete <em>all</em> documents in <em>my_doctype</em> schema:
+<pre>
+$ curl -X DELETE --cert data-plane-public-cert.pem --key data-plane-private-key.pem \
+  "$ENDPOINT/document/v1/my_namespace/my_doctype/docid?selection=my_doctype&amp;cluster=my_cluster"
```

Comment:
```suggestion
  "$ENDPOINT/document/v1/my_namespace/my_doctype/docid?selection=true&amp;cluster=my_cluster"
```

Using the selection `my_doctype` is not strictly required, as that is already implied by the REST path. I think it's better to use `selection=true` for this example. Feel free to disagree.